modules/claims_api/app/workers/claims_api/report_unsuccessful_submissions.rb

```diff
@@ -14,42 +14,57 @@ def perform
 
         ClaimsApi::UnsuccessfulReportMailer.build(@from, @to, consumer_totals: totals,
                                                               pending_submissions: pending,
-                                                              unsuccessful_submissions: errored_grouped,
+                                                              unsuccessful_submissions: unsuccessful_submissions,
+                                                              grouped_errors: errors_hash[:uniq_errors],
+                                                              grouped_warnings: errors_hash[:uniq_warnings],
```

Comment:
maybe this isnt done quite yet? but i dont see where these values are used in the view.

would be ideal to make sure this example email is working at the end of all this as well:
https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/mailers/previews/claims_api_unsuccessful_report_mailer_preview.rb

Reply:
ya no I ran out of time before I could finish the view changes, will be out today and I'll ask Dan to review since I know you're out.

Reply:
i dont see any changes to claims_api_unsuccessful_report_mailer_preview.rb and the preview still doesn't seem to work for me locally. is there a plan to get that functional? should probably either get it functioning at a basic level or remove it imo.

do you have any sort of screenshot of what this report is supposed to look like? i dont see one here or on the jira ticket but i definitely could have missed it. i was hoping the preview would be functional so i could check it out there but that doesnt seem to be the case.

Reply:
That's a good catch thanks, I'll get the previewer working.

Reply:
Done

Reply:
nice! looks great! thanks!